src/main/java/io/passport/server/model/Passport.java

```diff
@@ -35,5 +40,8 @@ public class Passport {
     @Column(name = "approved_by")
     private String approvedBy;
 
+    @Column(name = "details_json", columnDefinition = "jsonb")
+    @Convert(converter = JsonConverter.class)
+    @ColumnTransformer(write = "?::jsonb")
+    private Map<String, Object> detailsJson;
```

Comment:
Are we using this field to store information included in a Passport? Is it really necessary? I think instead, when the user generates a passport, we can save it to the file system and provide a REST endpoint where the user can access it. I suggest adding a TODO here to reflect this approach.